django_x509/static/django-x509/js/switcher.js

```diff
@@ -0,0 +1,42 @@
+(function($){
+    // function for operation_type switcher
+    var filter_div = function(operation_type){
+    // define fields for each operation_type
```

Comment:
indentation is not right here since you are defining a function you need to indent